gradle-ci.properties

```diff
@@ -29,10 +29,13 @@ org.gradle.configureondemand = true
 org.gradle.jvmargs = -Xmx4096m -XX:MaxMetaspaceSize=1024m
 
 # # Settings: Languages
-versions.java.language = 11
+versions.java.language = 17
+versions.java.target = 17
+versions.java.maximum = 19
```

Comment:
new platform minimum (driven by [this decision upstream](https://micronaut.io/2023/02/16/micronaut-framework-4-0-with-java-17-baseline/))